.restyled.yaml

```diff
@@ -1,7 +1,6 @@
 restylers_version: '20190904'
 auto: true
 restylers:
-  - prettier
```

Comment:
I removed it, because otherwise Restyled also reformats JavaScript files with `prettier`. The original commit message was “Restyled: Do not reformat JavaScript code” or something similar, if I remember correctly.

Reply:
I actually found it useful that JavaScript files are reformatted automatically. Any specific problem with this? (Except that our scripts currently do not do it.)

Reply:
> I actually found it useful that JavaScript files are reformatted automatically. Any specific problem with this? (Except that our scripts currently do not do it.)

Except from the last point you mentioned above, nope.

Reply:
so should I re-add it?

Reply:
It would be nice to have it but then the reformat- scripts should come to the same conclusion as the build server. So better to not do this within this PR.